src/cli/cli.hpp

```diff
@@ -339,6 +339,11 @@ class Interpreter : public OutputImplementer, public Utils
 
 #endif // OPENTHREAD_FTD || OPENTHREAD_MTD
 
+#if OPENTHREAD_CONFIG_DIAG_ENABLE
+    static void HandleDiagOutput(void *aContext, const char *aFormat, va_list aArguments);
```

Comment:
By convention, `aContext` is always the last parameter. Maybe let's keep it comsistent with other functions?